<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thermal Camera Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="image_label">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black; color: white;</string>
      </property>
      <property name="text">
       <string>No Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="edge_layout">
      <item>
       <layout class="QVBoxLayout" name="edge_buttons_layout">
        <item>
         <widget class="QPushButton" name="edge_mode_button">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Set Edge Off</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edge_auto_manual_button">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Set Auto Edge</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="t1_layout">
        <item>
         <widget class="QLabel" name="t1_label">
          <property name="text">
           <string>Threshold 1: 50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="t1_slider">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="t1_spinbox">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="t2_layout">
        <item>
         <widget class="QLabel" name="t2_label">
          <property name="text">
           <string>Threshold 2: 150</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="t2_slider">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="t2_spinbox">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="agc_layout">
      <item>
       <widget class="QPushButton" name="agc_mode_button">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Set Manual AGC</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="agc_min_layout">
        <item>
         <widget class="QLabel" name="agc_min_label">
          <property name="text">
           <string>Min Temp: 20.0°C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="agc_min_slider">
          <property name="minimum">
           <number>-500</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="agc_min_spinbox">
          <property name="minimum">
           <double>-50.0</double>
          </property>
          <property name="maximum">
           <double>500.0</double>
          </property>
          <property name="singleStep">
           <double>0.1</double>
          </property>
          <property name="value">
           <double>20.0</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="agc_max_layout">
        <item>
         <widget class="QLabel" name="agc_max_label">
          <property name="text">
           <string>Max Temp: 40.0°C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="agc_max_slider">
          <property name="minimum">
           <number>-500</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>400</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="agc_max_spinbox">
          <property name="minimum">
           <double>-50.0</double>
          </property>
          <property name="maximum">
           <double>500.0</double>
          </property>
          <property name="singleStep">
           <double>0.1</double>
          </property>
          <property name="value">
           <double>40.0</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="control_layout">
      <item>
       <widget class="QPushButton" name="connect_button">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_stream_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start Stream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_stream_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Stream</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
